File: samples/search/python/extract-search-features.py
import requests
from requests.auth import HTTPBasicAuth
import json

api_key = ""  # Your API key here, required
track_id = ""   # Track ID here, required

url = "https://api-us.musiio.com/v1/search/extract/search-features"

payload = {
    "id": track_id
}

headers = {
  'Content-Type': 'application/json'
}

response = requests.post(url, auth=HTTPBasicAuth(api_key, ''), headers=headers, data=json.dumps(payload))

print(response.text.encode('utf8'))

File: samples/search/node/extract-search-features.js
const API_KEY = ""; // Your API Key here
const TRACK_ID = "";// Track ID here

var axios = require('axios');
var data = JSON.stringify({"id": TRACK_ID});

var config = {
  method: 'post',
  url: 'https://api-us.musiio.com/v1/search/extract/search-features',
  auth: {
      username: API_KEY,
      password: ""
  },
  headers: { 
    'Content-Type': 'application/json'
  },
  data : data
};

axios(config)
.then(function (response) {
  console.log(JSON.stringify(response.data));
})
.catch(function (error) {
  console.log(error);
});
